tetris.py

```diff
@@ -117,12 +129,30 @@ def run_gui():
     global tetris_control
     tetris_control = TetrisControl()
 
+
+    global treeview
+    treeview = ttk.Treeview(game_frame, columns=('Time Spent', 'Game Speed', 'Score'), height=23)
+    treeview.tk.eval('''ttk::style theme use clam
+    ttk::style configure Treeview -fieldbackground gray7 -bordercolor royalblue''')
```

Comment:
Would be more readable like this:

```suggestion
    treeview.tk.eval('''
    ttk::style theme use clam
    ttk::style configure Treeview -fieldbackground gray7 -bordercolor royalblue
    ''')
```